server/cluster/coordinator.go

```diff
@@ -568,23 +565,20 @@ func (c *coordinator) removeScheduler(name string) error {
 	var err error
 	opt := c.cluster.opt
 	if err = opt.RemoveSchedulerCfg(s.Ctx(), name); err != nil {
-		log.Error("can not remove scheduler", zap.String("scheduler-name", name), zap.Error(err))
```

Comment:
Why remove these log?

Reply:
These logs will be printed on the upper level so I removed them to avoid duplication